README.md

```diff
@@ -127,6 +127,17 @@ with smart_hub.context("dashboard"):
         ... # your logic for each 'input<text/select/...>'
 ```
 
+### Limitation
+
+The "big" caveat here, is that we purposely don't use anything else than DOM analysis.
+No computer vision will be used in this project. We saw that introducing it is nice but unfortunately
```

Comment:
as a side note, [Maestro](https://docs.maestro.dev/api-reference/commands/assertwithai) seems to do visual testing

Reply:
Noted. Surely has the same weakness as what we already experienced.